subprojects/xcfa/xcfa-cli/src/main/java/hu/bme/mit/theta/xcfa/cli/params/ExitCodes.kt

```diff
@@ -32,6 +33,7 @@ enum class ExitCodes(val code: Int) {
     SERVER_ERROR(202),
     PORTFOLIO_ERROR(203),
 
+    UNSUPPORTED_ELEMENT(209),
```

Comment:
Another error code `UNSUPPORTED_BY_ANALYSIS` or similar could be useful as well, as unsupported language elements may differ on the analysis level.

Reply:
I agree, but this falls outside the scope of this current PR, as we currently do not keep track of language elements vs. backend capabilities.